clayui.com/plugins/gatsby-plugin-meta-redirect/gatsby-node.js

```diff
@@ -18,7 +18,10 @@ const writeRedirectsFile = async (redirects, folder, pathPrefix) => {
 	for (const redirect of redirects) {
 		const {fromPath, toPath} = redirect;
 
-		if (fromPath.endsWith('index.html')) {
+		if (
+			fromPath.endsWith('index.html') ||
+			fromPath.endsWith('contributing.html')
```

Comment:
This is the bit that makes you wince... maybe this could just be:

```js
if (fromPath.endsWith('.html')) {
```

Reply:
Yea, this is the one that made me insecure about my solution.